data/people.test.ts

```diff
@@ -24,7 +15,13 @@ const isValidPersonData = (person: Person): boolean => {
 describe('validate people.json', () => {
   it('필수 데이터가 모두 채워져 있어야 합니다.', () => {
     peopleJson.people.forEach(person => {
-      expect(isValidPersonData(person));
+      expect(isValidPersonData(person)).toBeTruthy();
+    });
+  });
+
+  it('허용된 year 데이터만 존재해야 합니다.', () => {
+    peopleJson.people.forEach(person => {
+      expect(VISIBLE_PEOPLE_YEARS_ORDER.includes(person.year)).toBeTruthy();
```

Comment:
jest에서 제공하는 matcher를 사용하면 더 좋은 실패 사유를 받을 수 있을 것 같아요.

```suggestion
      expect(VISIBLE_PEOPLE_YEARS_ORDER).toContain(person.year)
```

<img width="754" alt="image" src="https://github.com/AUSG/2022-homepage/assets/46379947/2a85636c-75fa-45f6-9ef0-91016e6eb845">
<img width="880" alt="image" src="https://github.com/AUSG/2022-homepage/assets/46379947/3dc88f5c-66f2-4ade-bc9e-acd5cbde575e">

Reply:
크으 jest 고수,, 훨씬 디테일하게 나오는군요! 이미지까지 굿굿 좋은 리뷰 감사합니다👍

Reply:
[5885545](https://github.com/AUSG/2022-homepage/pull/85/commits/58855456986aefb0035a778f9613bb631676acca)